avocado/core/job.py

```diff
@@ -131,6 +131,7 @@ def __init__(self, args=None):
         #: has not been attempted.  If set to an empty list, it means that no
         #: test was found during resolution.
         self.test_suite = None
+        self.test_runner = None
```

Comment:
OK, this didn't look like a pylint only fix, but indeed, it's a good idea to initialize it to None.